README.md

```diff
@@ -10,73 +10,74 @@ Infrastructure to watch a CloudWatch log group and forward its logs to Humio.
+| humio_lambda_log_retention                | number       | Number of days to retain CloudWatch logs from the Humio Lambda functions.                                                                                                                    | 1       |
+| enable_cloudwatch_logs_auto_subscription  | bool         | Make the log ingester automatically subscribe to new log groups specified with the logs subscription prefix parameter. Set to 'true' to enable.                                              | false   |
+| humio_cloudwatch_logs_subscription_prefix | string       | Humio will only subscribe to log groups with the prefix specified.                                                                                                                           | ""      |
+| enable_cloudwatch_logs_backfiller_autorun | bool         | Make the backfiller run automatically when created. Set to 'true' to enable.                                                                                                                 | false   |
+| vpc_id                                    | string       | Use a VPC for the lambda ingester functions. Pass in a vpc to enable.                                                                                                                        |         |
+| security_group_ids                        | list(string) | A list of security group ids for the VPC configuration regarding the ingester lambda functions. Only required if VPC is enabled.                                                             |         |
+| subnet_ids                                | list(string) | A list of subnet ids used by the VPC configuration that the ingester lambda functions will be deployed into. Only required if VPC is enabled.                                                |         |
+| humio_lambda_log_level                    | string       | The log level for the Humio lambdas. (DEBUG, INFO, WARNING, ERROR, CRITICAL)                                                                                                                 | "INFO"  |
+| cloudwatch2humio_version                  | string       | The version of the integration to be installed. When creating a new stack, the default value is the newest version. Available releases can be found under releases in the GitHub repository. |
 
 ## Outputs
 
-| Name | Type | Description |
-| ---  | ---  | --- |
-| function_name | string | The name of the logs-to-Humio function. |
+| Name                                      | Type         | Description                                                                                                                                                                                  | Default |
```

Comment:
why do the outputs have a default column?

Reply:
TBH I didn't see the default column until you just mentioned it. With each table entry on one line, I couldn't see the column on my screen unless I scrolled waaaaay to the right 😅